src/quo/components/avatars/community_avatar/schema.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo.components.avatars.community-avatar.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:size {:optional true} [:maybe [:enum :size-32 :size-24]]]
+      [:image :schema.common/image-source]
+      [:container-style {:optional true} [:maybe :map]]]]]
+   :any])
```

Comment:
The schema can be in the same ns with the component if it's small like this one. We tend to move it to a separate ns if you end up composing the schema from other smaller schema definitions, which takes some space and pollutes the component ns.

Reply:
Personally, I prefer keeping schemas in a separate file. In my workflow, I find it very easy to locate and work with component schemas when they are defined separately. However, I don't have strong objections to writing small schemas within the component view file itself. It seems we still haven't reached a consensus on this.